examples/vdpa/vdpa_rpc.h

```diff
@@ -0,0 +1,40 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright 2022, NVIDIA CORPORATION & AFFILIATES.
+ */
+
+#ifndef _VDPA_RPC_H_
+#define _VDPA_RPC_H_
+
+#include <getopt.h>
+#include <signal.h>
+#include <stdint.h>
+#include <string.h>
+#include <unistd.h>
+
+#include <jsonrpc-c.h>
+
+#define MAX_PATH_LEN 128
+/* VDPA RPC */
+#define VIRTNET_FEATURE_SZ	64
+#define MAX_VDPA_SAMPLE_PORTS 1024
+#define RTE_LOGTYPE_RPC RTE_LOGTYPE_USER1
+#define VDPA_LOCAL_HOST "127.0.0.1"
+
+enum {
+	VDPA_RPC_PORT = 12190,
```

Comment:
Could user change this in runtime?

Reply:
No, it is hardcode. It is exampler for customer. They can change it in his code